ofctrl/ofctrl.go

```diff
@@ -257,22 +257,21 @@ func (c *Controller) handleConnection(conn net.Conn) {
 
 	for {
 		select {
-		// Send hello message with latest protocol version.
```

Comment:
Why delete this comment?

Reply:
The comment was not placed correctly as the HELLO message was sent few lines up where there is already a comment.